casanovo/casanovo.py

```diff
@@ -1,8 +1,13 @@
 """The command line entry point for Casanovo."""
+import collections
+import csv
 import datetime
 import logging
 import os
+import pathlib
```

Comment:
**question:** Typically, pathlib is used as `from pathlib import Path`. Is there a particular reason to break that convention here? [docs](https://docs.python.org/3/library/pathlib.html#basic-use)